lib/taskana-core/src/main/java/pro/taskana/spi/task/api/AfterRequestChangesProvider.java

```diff
@@ -0,0 +1,43 @@
+package pro.taskana.spi.task.api;
+
+import pro.taskana.common.api.TaskanaEngine;
+import pro.taskana.task.api.models.Task;
+
+/**
+ * The AfterRequestChangesProvider allows to implement customized behaviour after changes have been
+ * requested on a given {@linkplain Task}.
+ */
+public interface AfterRequestChangesProvider {
+
+  /**
+   * Provide the active {@linkplain TaskanaEngine} which is initialized for this TASKANA
+   * installation.
+   *
+   * <p>This method is called during TASKANA startup and allows the service provider to store the
+   * active {@linkplain TaskanaEngine} for later usage.
+   *
+   * @param taskanaEngine the active {@linkplain TaskanaEngine} which is initialized for this
+   *     installation
+   */
+  void initialize(TaskanaEngine taskanaEngine);
+
+  /**
+   * Perform any action after have been requested on a {@linkplain Task} through {@linkplain
+   * pro.taskana.task.api.TaskService#requestChanges(String)}.
```

Comment:
Something is missing in this description.

Perform any action after [a review] ha[s] been requested

Reply:
I will complete the description :)